.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: java
 jdk: oraclejdk8
 dist: trusty
-
-# Make the gradle wrapper executable
-before_install: chmod +x gradlew
+sudo: false
```

Comment:
Not sure if you already told me somewhere (sorry if you did) but why would this need to be changed? At the article at https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration they say 

> If you currently specify sudo: false in your .travis.yml, we recommend removing that configuration soon.

Reply:
Really?! Probably this invalidates my pr. I'm closing it.